compiler/ast/ast_parsed_types.nim

```diff
@@ -184,8 +183,9 @@ type
     pnkIteratorDef
     pnkMacroDef
     pnkTemplateDef
-    pnkTypeSection
+    pnkConstDef
     pnkTypeDef
+    pnkIdentDefs
```

Comment:
This should appear before `pnkConstDef` and `pnkTypeDef`, a range for these would read awkwardly otherwise `{pnkConstDef .. pnkIdentDefs}` -- the important part is that they're all ident definitions.

Reply:
`pnkVarTuple` also needs to be considered here, as it's also an identifier definition.